server/build-plugins.js

```diff
@@ -18,6 +18,15 @@ const camelize = str =>
     })
     .replace(/[^\w]/gi, '');
 
+const getPackageName = name => {
+  if (name.indexOf('/') !== -1 && name[0] !== '@') {
```

Comment:
npm packages don't necessarily need to start with an @. If someone has published a plugin independently and it's not scoped then it would just appear as a normal npm package. Official bPanel plugins will be scoped with `@bpanel`, but others published by independent devs won't necessarily.

Reply:
If you're trying to download a scoped package from `npm`, it will have an @. From the [npm docs](https://docs.npmjs.com/cli/install):

```Note: if you do not include the @-symbol on your scope name, npm will interpret this as a GitHub repository instead, see below. Scopes names must also be followed by a slash.```

That's why it checks for both the slash and the absence of an @.

Reply:
ah, ok! Gotcha. I think i just misread the conditional 😊